src/crypto/sha256_power8.cpp

```diff
@@ -0,0 +1,405 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+//
+// This is a translation to GCC extended asm syntax from YASM code by Intel
+// (available at the bottom of this file).
+
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#ifdef HAVE_POWER8
```

Comment:
Since this file is only compiled with `HAVE_POWER8`, I think we don't need this here.